EpsilonCCS/Src/TelemetryReporting/TelemetryReporting.c

```diff
@@ -82,7 +84,57 @@ void sendKeyMotor()
 
 void sendMotorDetails(int n)
 {
+    unsigned int unframedPacketLength = MOTOR_DETAILS_LENGTH + CHECKSUM_LENGTH;
+    unsigned char packetPayload[unframedPacketLength];
+
+    switch (n)
+    {
+        case 0:
+            packetPayload[0] = M0_DETAILS_DATA_PKG_ID;
+            writeFloatIntoArray(packetPayload, 1, motor0DetailsData.phaseCCurrent);
+            writeFloatIntoArray(packetPayload, 5, motor0DetailsData.phaseBCurrent);
+            writeFloatIntoArray(packetPayload, 9, motor0DetailsData.motorVoltageReal);
+            writeFloatIntoArray(packetPayload, 13, motor0DetailsData.motorVoltageImaginary);
+            writeFloatIntoArray(packetPayload, 17, motor0DetailsData.motorCurrentReal);
+            writeFloatIntoArray(packetPayload, 17, motor0DetailsData.motorCurrentImaginary);
```

Comment:
Looks like you missed bumping the index by 4 here. Same for motor 1.

Bump all indices below this by 4 as well. Last one should be at 65.